pkg/scheduler/util/heap.go

```diff
@@ -240,12 +241,12 @@ func (h *Heap) Len() int {
 }
 
 // NewHeap returns a Heap which can be used to queue up items to process.
-func NewHeap(keyFn KeyFunc, lessFn LessFunc) *Heap {
+func NewHeap(keyFn KeyFunc, lessFn podinfo.LessFunc) *Heap {
```

Comment:
We should move LessFunc to this file IMO.

Reply:
@draveness Can you confirm the changes?